packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -1,34 +1,37 @@
 import React, {
-  useContext, useRef, useCallback,
+  useContext, useRef, useState, useEffect,
 } from 'react';
+import './_elementPolyfill';
 import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
 import classNames from 'classnames/bind';
 import ThemeContext from 'terra-theme-context';
-import { injectIntl } from 'react-intl';
 import * as KeyCode from 'keycode-js';
-import ColumnHeaderCell from './ColumnHeaderCell';
-import WorklistDataGridPropTypes from './proptypes/WorklistDataGridPropTypes';
-import './_elementPolyfill';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
 import styles from './WorklistDataGrid.module.scss';
+import WorklistDataGridUtils from './utils/WorklistDataGridUtils';
+import WorklistDataGridPropTypes from './proptypes/WorklistDataGridPropTypes';
+import Row from './subcomponents/Row';
+import ColumnHeaderCell from './subcomponents/ColumnHeaderCell';
 
 const cx = classNames.bind(styles);
 
 const propTypes = {
   /**
-   * String representing an attribute which identifies the element (or elements) that labels the grid.
```

Comment:
Why was 'or elements' removed? `aria-labelledby` can take a list of ids (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby#benefits_and_drawbacks), unless we want to enforce that only one element may be provided.

Reply:
Restored the comment.